less/sequences.less

```diff
@@ -131,7 +131,7 @@
 
     &:after,
     ~ li:after {
-      background-color: @sequence-border;
+      background-color: @sequence-border-after;
```

Comment:
Hey George, why is the colour called border if it's for the bg?

Reply:
A great question. I'll change that.